robot-core/src/main/java/org/obolibrary/robot/TemplateOperation.java

```diff
@@ -365,11 +341,11 @@ public static OWLOntology template(
         if (template.isEmpty()) {
           continue;
         }
-        if (!validateTemplateString(template)) {
-          throw new Exception(
-              String.format(
-                  unknownTemplateError, tableName, column + 1, headers.get(column), template));
-        }
+        // if (!validateTemplateString(template)) {
```

Comment:
If this code isn't used, don't just comment it out, delete it. The same goes for the commented-out code below.